README.md

```diff
@@ -50,6 +50,13 @@ Here are some specific feature highlights of Grist:
     - Control access based on cell values and user attributes.
   * Can be self-maintained.
     - Useful for intranet operation and specific compliance requirements.
+  * Several sandboxing options for untrusted documents.
+    - On Linux or with docker, you can enable
+	  [gVisor](https://github.com/google/gvisor) sandboxing at the individual
+	  document level.
+    - On OSX, you can use native sandboxing.
+	- On any OS, if running Grist outside of docker, you can use docker
```

Comment:
No. If running Grist outside of docker (for example when compiled from source), there is a sandboxing flavor that will run individual data engines in docker containers. That flavor isn't available when running Grist itself in docker, since running docker within docker is something that cannot be done by default.

Reply:
I could just remove mention of it since it is confusing for sure, and not polished at all?

Reply:
Removed mention of this mode.

Reply:
But when you run with `docker run -p 8484:8484 -it gristlabs/grist`, it does run in a docker container, correct ? Does this form provide some sort of sandboxing ?

Reply:
It does limit access to the host, yes. Formulas could access any Grist document (or file) within the container, and also would have network access.

Reply:
Thank you for the clarification, it makes perfect sense.